src/containers/DashboardWrapper/BottomMenu/index.tsx

```diff
@@ -95,17 +94,57 @@ function Footer({ className, history }: Props): JSX.Element {
         [choice, setDashboard],
     )
 
+    const menuRef = useRef<HTMLDivElement>()
+
+    const [mobileWidth, setMobileWidth] = useState<boolean>(
+        document.body.clientWidth <= 900,
+    )
+    const width = useWindowWidth()
+    useEffect(() => {
+        if (width > 900 && mobileWidth) {
+            setMobileWidth(false)
+            menuRef.current.style.transform = ''
```

Comment:
Mulig det blir litt ryddigere å ha useState() for transform og så bruke inline CSS.

Reply:
Kan godt hende det er mulig å gjøre dette med media queries også, men vet ikke om det blir renere om man må knote rundt med `!important`eller spesifisitet 🤷‍♂️

Reply:
Tror jeg lar denne være